packages/neutron-sdk/src/interchain_queries/helpers.rs

```diff
@@ -1,8 +1,12 @@
 use crate::errors::error::{NeutronError, NeutronResult};
 use crate::interchain_queries::types::{
-    AddressBytes, BALANCES_PREFIX, DELEGATION_KEY, MAX_ADDR_LEN, PARAMS_STORE_DELIMITER,
-    VALIDATORS_KEY,
+    AddressBytes, BALANCES_PREFIX, DELEGATION_KEY, FEE_POOL_KEY, MAX_ADDR_LEN,
+    PARAMS_STORE_DELIMITER, PROPOSALS_KEY_PREFIX, SUPPLY_PREFIX, VALIDATORS_KEY,
 };
+use cosmos_sdk_proto::cosmos::staking::v1beta1::Commission as ValidatorCommission;
+use cosmwasm_std::{Decimal, Uint128};
+use std::str;
+use std::str::FromStr;
```

Comment:
You only import `std::str` for `from_utf8` symbol.
I suggest restructuring imports in this way:
```suggestion
use std::str::{from_utf8, FromStr};
```